backend/tests/setup.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-console */
+// global.console = {
+//   ...global.console,
+//   log: jest.fn(), // console.log are ignored in tests
```

Comment:
Perhaps we should keep this to suppress `console.log` in tests to keep the output more tidy.